outofcore/include/pcl/outofcore/octree_base.h

```diff
@@ -612,21 +612,18 @@ namespace pcl
         checkExtension (const boost::filesystem::path& path_name);
 
 
-        /** \brief DEPRECATED - Flush all nodes' cache 
-         *  \deprecated this was moved to the octree_node class
-         */
+        /** \brief Flush all nodes' cache */
+        PCL_DEPRECATED(1, 12, "this was moved to the octree_node class")
```

Comment:
I checked the implementation of this method and I believe preserving thi method is still worth it. This object holds a tree with nodes and flushToDisk is something which takes place at node level and is propagated to the child nodes. However, having the same method in the container ensures this is automatically called from the root node. This method is useful, let's just remove the deprecation notice.